src/fancyline/tty/vt100.cr

```diff
@@ -2,16 +2,23 @@ class Fancyline
   class Tty
     lib LibC
       struct Winsize
-        ws_row : UInt16    #  rows, in characters
-        ws_col : UInt16    #  columns, in characters
-        ws_xpixel : UInt16 #  horizontal size, pixels
-        ws_ypixel : UInt16 #  vertical size, pixels
+        ws_row    : ::LibC::UShort # rows, in characters
+        ws_col    : ::LibC::UShort # columns, in characters
+        ws_xpixel : ::LibC::UShort # horizontal size, pixels
+        ws_ypixel : ::LibC::UShort # vertical size, pixels
       end
 
+      {% unless flag?(:x86_64) && flag?(:darwin)%}
```

Comment:
Instead of a single `unless`, use an inclusive `if/elsif` tree.  Makes it simple to be extended later on:

```cr
{% if flag?(:x86_64) && flag?(:darwin) %}
  # Darwin constants
{% else %}
  {% puts "Warning: ..." %}
{% end %}
```

Becomes quite easy with that to e.g. add Linux@x86_64 later on.